libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/changeSharingHandler.ts

```diff
@@ -81,6 +97,13 @@ function* saveSharing(
         saveSharingCtx,
     );
 
+    if (saveSharingCtx.cmd.payload.newShareProps.granteesToAdd.length !== 0) {
```

Comment:
Detail: in case there are both grantees to add and delete, you could run those two requests in parallel.

Reply:
We will let it as it is.